docs/development.md

```diff
@@ -36,53 +36,95 @@ This is building Epinio for linux on amd64 architecture. If you are on a
 different OS or architecture you can use one of the available `build-*` targets.
 Look at the Makefile at the root of the project to see what is available.
 
-
 ### Installing Epinio
 
 You can have a look at [the dedicated document](/docs/install.md) for cluster
 specific instructions, but generally this should be sufficient to get you running:
 
 ```
-./dist/epinio-linux-amd64 install
+export EPINIO_DONT_WAIT_FOR_DEPLOYMENT=1
```

Comment:
I suppose it is irrelevant if we use `true` or `1`?

Reply:
Would have to check.

Reply:
Checked: Any non-empty string is sufficient.
https://github.com/epinio/epinio/blob/main/deployments/epinio.go#L195